src/sentry/static/sentry/app/components/clippedBox.jsx

```diff
@@ -42,8 +42,21 @@ class ClippedBox extends React.Component {
     }
   }
 
+  componentDidUpdate() {
+    if (this.state.revealed || !this.state.clipped) {
+      return;
+    }
+
+    const renderedHeight = ReactDOM.findDOMNode(this).offsetHeight; // eslint-disable-line react/no-find-dom-node
+    if (renderedHeight < this.props.clipHeight) {
+      this.reveal();
+    }
+  }
+
   reveal = e => {
-    e.stopPropagation();
+    if (e) {
+      e.stopPropagation();
```

Comment:
maybe you could change it to `e?.stopPropagation()` ?

Reply:
yes! 🔥